cert/lib/cert/options.rb

```diff
@@ -13,6 +13,16 @@ def self.available_options
                                      description: "Create a development certificate instead of a distribution one",
                                      is_string: false,
                                      default_value: false),
+        FastlaneCore::ConfigItem.new(key: :generate_p12,
+                                     env_name: "CERT_GENERATE_P12_FILE",
+                                     description: "Generate a p12 file additionally to a PEM file",
+                                     is_string: false,
+                                     default_value: true),
+        FastlaneCore::ConfigItem.new(key: :p12_password,
+                                     env_name: "CERT_P12_PASSWORD",
+                                     sensitive: true,
+                                     description: "The password that is used for your p12 file",
+                                     default_value: ""),
```

Comment:
since `:generate_p12` is `true` by default wouldn't that fail/corrupt the p12 file because of `blank password` ?